frontend/src/js/services/stringToTemplate.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Created by aayusharora on 8/19/17.
```

Comment:
@aayusharora: Hey. This is nice of you but we should not write our personal details in code files. However you can raise an issue to add CONTRIBUTORS.md in the root directory.